ts/private/ts_project.bzl

```diff
@@ -219,14 +219,21 @@ This is an error because Bazel does not run actions unless their outputs are nee
             transitive = transitive_inputs + [_gather_declarations_from_js_providers(ctx.attr.srcs + [ctx.attr.tsconfig] + ctx.attr.deps)],
         )
 
+        verb = "Type-checking & Transpiling"
```

Comment:
nit: Transpiling & type-checking

would be my preference

Reply:
👍